priv/worker.py

```diff
@@ -200,7 +216,7 @@ def process_response(response):
   global envs
   global compressionLevel
 
-  response = str(response)
+  response = str(response, "utf-8")
```

Comment:
This doesn't work for python2.7, but I think we can use: ``response = unicode(response, "utf-8")``here.